akka-docs/src/main/paradox/general/actor-systems.md

```diff
@@ -115,8 +115,8 @@ while Akka does the heavy lifting under the hood.
 ## Terminating ActorSystem
 
 When you know everything is done for your application, you can call the
-`terminate` method of `ActorSystem`. That will stop the guardian
-actor, which in turn will recursively stop all its child actors, the system
+`terminate` method of `ActorSystem`. That will run @ref:[`CoordinatedShutdown`](../actors.md#coordinated-shutdown)
+followed by stopping the guardian actor, which in turn will recursively stop all its child actors, the system
 guardian.
```

Comment:
This too makes it sound like the system guardian is the primary child of the guardian, which doesn't seem to be the case?